src/test/kotlin/nevam/MavenCoordinatesTest.kt

```diff
@@ -18,4 +21,18 @@ class MavenCoordinatesTest {
     assertThat(coordinates.mavenDirectory(includeVersion = false)).isEqualTo("com/squareup/sqldelight/runtime")
     assertThat(coordinates.mavenDirectory(includeVersion = true)).isEqualTo("com/squareup/sqldelight/runtime/1.4.0")
   }
+
+  @Test fun `read properties from file with all properties succeeds`() {
+    val coordinates = MavenCoordinates.readFrom("src/test/resources/full.gradle.properties")
+    assertThat(coordinates.toString()).isEqualTo("com.squareup.sqldelight:runtime:1.4.0")
+  }
+
+  @Test fun `read properties from file with missing properties suggests -c`() {
+    try {
+      MavenCoordinates.readFrom("src/test/resources/missing.gradle.properties")
+      fail("Expected a CliktError suggesting -c")
```

Comment:
Can `assertThrows` be used here instead of using an exception as a failsafe?

Reply:
I'm not familiar with `assertThrows`. Googling tells me it's in JUnit 5 but this project uses JUnit 4. Is there another library that has it that you want me to use?

Alternatively I could do `@Test(expected = CliktError::class)` but then I can't verify the message.

Happy to do it whatever way you prefer, let me know!

Reply:
Ah right. I should try out junit 5 for startship but that's outside the scope. What do you think of using `ExpectedException` rule here? If it's too much of an effort then let's go ahead with this for now.

Reply:
Done!